to-json-schema.js

```diff
@@ -70,6 +88,50 @@ const exampleAttributeMapping = (typeInput, example, jsonSchemaInput) => {
   }
 };
 
+const additionalAttributesMapping = (typeInput, avroDefinition, jsonSchemaInput) => {
+  const __ret = extractNonNullableTypeIfNeeded(typeInput, jsonSchemaInput);
+  const type = __ret.type;
+  const jsonSchema = __ret.jsonSchema;
+
+  exampleAttributeMapping(type, avroDefinition.example, jsonSchema);
+
+  function setAdditionalAttribute(name) {
+    let isValueCoherent = true;
+    if (name === 'minLength' || name === 'maxLength') {
+      isValueCoherent = avroDefinition[name] > -1;
+    } else if (name === 'multipleOf') {
+      isValueCoherent = avroDefinition[name] > 0;
+    }
+    if (avroDefinition[name] !== undefined && isValueCoherent) jsonSchema[name] = avroDefinition[name];
+  }
+
+  switch (type) {
+  case 'int':
+  case 'long':
+  case 'float':
+  case 'double':
+    setAdditionalAttribute('minimum');
+    setAdditionalAttribute('maximum');
+    setAdditionalAttribute('exclusiveMinimum');
+    setAdditionalAttribute('exclusiveMaximum');
+    setAdditionalAttribute('multipleOf');
```

Comment:
Subjective style idea - a `setAdditionalAttributes` helper function that takes an array of strings would help simplify this bit. 

```js
setAdditionalAttributes(['minimum', 'maximum', 'exclusiveMinimum', ...
```